charters/DataProcessingPipelines/charter.md

```diff
@@ -0,0 +1,98 @@
+* Reviewing community tools for use in data processing pipelines  
+* Working with the community to provide feedback on metadata standards for raw data in the HCA DCP to provide the required information to enable processing of the raw data  
+* Evaluating and adopting file formats and associated standards for outputs of pipelines, ensuring the data is accessible with standard tools and interoperable with downstream processes  
+
+### Pipeline Benchmarking
+
+* Investigating and identifying data for pipeline benchmarking  
+* Establishing and operating automated practices around scientific testing of pipelines  
+* Establishing standard benchmarking used to evaluate data processing pipelines  
+
+### Pipeline Improvement
+
+* Updating pipelines ran in the HCA DCP for optimization of cost, time, or scientific benchmarks  
+
+### Support
+
+* Performing outreach to users on the usability of data in downstream applications  
+* Performing outreach to developers on pipeline development in the HCA DCP  
+* Working with the metadata team and data consumers to ensure the metadata provided about pipelines and the analysis results meet the needs of downstream users  
+* Creating tools that support the creation of community pipelines - may include software, documentation or support  
+
+## Scientific "guardrails"
+
+* Initial versions of standardized pipelines must be reviewed by the AWG.  
+* Updates to standardized pipelines that change the scientific output must be reviewed by the AWG. Engineering updates that do not change the scientific output do not require review by the AWG.  
+* Selection of standardized pipelines to develop must be approved by the HCA DCP Governance Group (Note, this does not apply to Community Pipelines which are onboarded as a service to the scientific community). The Data Processing Pipelines team is responsible for setting the development priorities for the selected pipelines.  
+
+## Out of Scope
+
+* Educational training or user assistance on how to perform data analysis with outputs of pipelines  
+* Recommendations or demonstrations of downstream tool use beyond initial data loading  
+* Novel methodological development is not performed by the Data Processing Pipelines team unless it is essential to the pipeline, it is not available in community tools, and requires minimal effort to implement  
+* The Data Processing Piplines team preferences using 3rd party software from the community but will not maintain or perform long-term engineering for those tools.  
+
+## Milestones and Deliverables
+
+__2018:__ Two data processing pipelines in production (Smart-seq2 and 10xv2 human data)  
+__2019:__ Adding at least three more data processing pipelines, including 10xv2 snRNA-seq and mouse pipelines for all current analysis pipelines Aim to have at least one _in situ_ transcriptomics pipeline in this year. Aim for one of these pipelines to be a community pipeline. Improved automated testing and benchmarking of analysis pipelines  
```

Comment:
```suggestion
__2019:__ Adding at least three more data processing pipelines, including 10xv2 snRNA-seq and mouse pipelines for all current analysis pipelines Aim to have at least one _in situ_ transcriptomics pipeline in this year. Aim for one of these pipelines to be a community pipeline. Improved automated testing and benchmarking of analysis pipelines  
```

Reply:
Sorry - hit the wrong button. Given that we will be discussing pipeline prioritization with the GG on the December 13th call, do others agree that we should wait to solidify these priorities/deliverables until we have their feedback and are able to integrate it? Perhaps even add a designee of the AWG as a reviewer?

Reply:
Given that both of you feel that the GG feedback would be valuable, I would recommend treating this as a variation of the normal  _Science Review_ flow described in the charter process:

- Assign the Science Reviewer as a reviewer of the pull request
- Add the "science-review-required" label
- Add a two week last call for the completion of the Science review to the top-level summary comment in the charter pull request.

EXAMPLE
**April 5**: Last call for science review

- When the feedback from the Science review is addressed, replace the "science-review-required" label with "science-review-completed".

@jonahcool - It's also an intriguing proposal to add an AWG member as reviewer.

Reply:
We could do one of two things, confirm our priorities with the GG and make sure the charter is inline after the meeting or ask a GG member for review. Given it is pipeline prioritization it should be the GG reviewing.